inboxes/urls.py

```diff
@@ -4,9 +4,10 @@
 from django.contrib.auth import views as auth_views
 
 from . import views
-from .views import PostListView
+from .views import PostListView, OneToOneView
 
 app_name = 'inboxes'
 urlpatterns = [
     path('feed/', PostListView.as_view(), name='postList'),
+    path('1to1/', OneToOneView.as_view(), name='1to1'),
```

Comment:
I think it might be best to use `one_to_one` instead of 1to1. Especially if all the classes and methods are called OneToOne